doc/md/base/Region.md

```diff
@@ -109,6 +109,8 @@ Every new page is zero-initialized, containing byte 0x00 at every offset.
 Function `grow` is capped by a soft limit on `size` controlled by compile-time flag
  `--max-stable-pages <n>` (the default is 65536, or 4GiB).
 
+There is currently no way to shrink the Region size.
+
```

Comment:
```suggestion
There is currently no way to shrink the Region size.

```

Reply:
This is an auto generated file - I'll instead modify the source herehttps://github.com/dfinity/motoko-base/pull/632